imports/plugins/included/payments-stripe/server/no-meteor/i18n/en.json

```diff
@@ -3,7 +3,7 @@
   "i18n": "en",
   "ns": "reaction-stripe",
   "translation": {
-    "reaction-payments": {
+    "reaction-stripe": {
```

Comment:
Same comment for all translation files in this package: Should this be `reaction-payments-stripe` or just `payments-stripe` to match the package name?

Reply:
It's supposed to match `ns` in this same file, so that's why I changed it. There were a few files that were probably copied and were not validly formatted, which didn't matter with the way we were loading before, but now caused errors.

I didn't give much thought to what the namespace _should_ be other than fixing the mismatch bug. I think the general rule would probably be to match identically the package name, but our package names are also a bit inconsistent.

@mikemurray Do you foresee any problems if I audit these namespaces and ensure they exactly match the package names?

Reply:
I think it should be fine to make them match the package. As far as I remember, the name is supposed to match the package anyway, and things may have gotten out of sync with all the copy-pasting of translations around the app.

Reply:
After reviewing all, they actually do all match the package names. So we should try to improve the consistency of the package names, but that's something I won't tackle here.